### DISCUSSION TITLE
Separate limit for syncing full and partial backups.

### DISCUSSION THREAD

**Discussion by kurdybanek** [lavinir/hassio-onedrive-backup#68] [Ideas]
I see that If I enable syncing of other backups, they are syncing regardless if they were full or partial backups. It would be nice to have ability for more fine-tunning, like syncing other backups, but only if they are full ones. 
Or (maybe the better idea) to have two limits, one for partial and one for full backups.

Isee two scenarios here, for which it would be helpful:
1. If you do a lot of changes (like upgrades, installin/removing addons) it may result with lot of partial backups (the default for HA is to do partial backup with every change like that). Then you may end up with having only partial backups on your onedrive, if the limit is set to relatively low number (default is 2 as I remember).

2. You may increase the number of backups stored on onedrive, which will be solution for the case 1, but this may result in consuming large amount of storage space when you do not do much changes for longer period of time. As a result you will end up with large number of full backups, not necessarily helpful, as there were no much changes in between.

Having two limits will give opportunity to have both birds at the same time - save space by limiting the number of full backups, and have better flexibility of choosing the proper backup to restore if you do many changes at once.

**Comment by lavinir** [comment]
Not sure I agree with the distinction between full / partial. 
You can set your backups so that all of them are partial  (e.g. - omit the /media folder, ignore a specific addon etc.) and in fact I know many users including myself only perform partial backups. 

With this in mind the separation doesn't really make sense and would only complicate the addon configuration.
